### ISSUE TITLE
[BUG] When a backport fails the github action should fail

### ISSUE THREAD

**Issue by peternied** [VachaShah/backport#9]
When the backport doesn't work, the github action results don't reflect that the backport didn't work correctly.  It would be useful to find backport failures via inspection of the backport github action job results.
```
....
  hint: You can instead skip this commit with "git cherry-pick --skip".
  hint: To abort and get back to the state before "git cherry-pick",
  hint: run "git cherry-pick --abort".
  /usr/bin/git cherry-pick --abort
  Error: Error: The process '/usr/bin/git' failed with exit code 1
```

https://github.com/opensearch-project/security/actions/runs/5177797123/jobs/9328356820

**Comment by peternied** [comment]
It looks like after a backport fails, we write out a commit message, but don't propagate a failure up to the action, see https://github.com/VachaShah/backport/blob/a82e65ee424186d5328291321945b2444271af1f/src/backport.ts#L324

**Comment by AMoo-Miki** [comment]
A failure of an Action is not so easy to trace (specially considering our workflow where every PR is backported to several branches) but if a failing backport results in the original PR being labelled with something like a `failed backport`, maintainers can just search PRs for that label and take action. I have a PR, #8, for this specific thing.

**Comment by peternied** [comment]
That is even a better solution that what I proposed, nice work!

**Comment by VachaShah** [comment]
PR #8 is merged and released in version 2.2.0.
